sdk/python/feast/repo_config.py

```diff
@@ -43,9 +50,14 @@ class Config:
 class RegistryConfig(FeastBaseModel):
     """ Metadata Store Configuration. Configuration that relates to reading from and writing to the Feast registry."""
 
-    path: StrictStr
+    path: StrictStr = "data/"
     """ str: Path to metadata store. Can be a local path, or remote object storage path, e.g. a GCS URI """
 
+    publish_json: StrictBool = False
+
+    is_filepath: StrictBool = False
```

Comment:
Is there a reason to ask users to specify this configuration flag instead of handling the migration for them? We know if we are being provided a file location vs a directory, right?

Reply:
Good point, let me change it to validate file vs directory